src/screens/studentGroup.tsx

```diff
@@ -126,9 +133,21 @@ const StudentGroup: React.FC = () => {
       .catch(err => console.log(err))
   }
 
+  const backHandler = () => {
+    history.push('/')
+  }
+
   return (
     <>
       <Box display="flex" flexDirection="column" alignItems="center">
+        <IconButton
+          icon={<ArrowBackIcon />}
+          aria-label="back"
+          position="absolute"
```

Comment:
is this absolutely positioned on the page or relative to its parent?

Reply:
I think it's absolute on the page.  Is that bad?

Reply:
yeah sounds brittle, what are you trying to accomplish

Reply:
Just to have the back button near the top left of the screen

Reply:
yeah just pulled down the code, i'd put the title and the button in a div, set the div to position relative and position the button absolutely relative to that, the way you're doing it here that back button will remain in that position no matter what, we don't really know what we're doing with respect to the layout yet

Reply:
Okay, awesome.  I'll do that.

Reply:
Finally got around to fixing this